boot/zephyr/Kconfig

```diff
@@ -76,19 +87,28 @@ config BOOT_SIGNATURE_TYPE_ECDSA_P256
 if BOOT_SIGNATURE_TYPE_ECDSA_P256
 choice
 	prompt "Ecdsa implementation"
+	default BOOT_SHARED_CRYPTO if SECURE_BOOT
 	default BOOT_CC310 if HAS_HW_NRF_CC310
 	default BOOT_TINYCRYPT
+
 config BOOT_TINYCRYPT
 	bool "Use tinycrypt"
 	select BOOT_USE_TINYCRYPT
+
 config BOOT_CC310
 	bool "Use CC310"
 	select BOOT_USE_NRF_CC310_BL if HAS_HW_NRF_CC310
-	select NRF_CC310_BL if HAS_HW_NRF_CC310
 	select NRFXLIB_CRYPTO if SOC_FAMILY_NRF
 	select BOOT_USE_CC310
+
+config BOOT_SHARED_CRYPTO
```

Comment:
Maybe this should be named BOOT_EXTERNAL_CRYPTO - form point of view of bootloader it uses external component. With shared it is always not clear who sheared something.

Reply:
If this will be changed - all configs/macros should align names.

Reply:
`BOOT_NRF_EXTERNAL_CRYPTO` ? or are you for not keeping any `NRF_` specific prefix onto things in this repository? with regards to https://github.com/NordicPlayground/fw-nrfconnect-mcuboot/pull/18#pullrequestreview-222252013

Reply:
Can contain `NRF` until nobody else have them-own need.

Reply:
This is now renamed